CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## v1.6.10
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Istio integration has been updated to release 1.5.0. (Backport PR #12356, Upstream PR #10564, @jrajahalme)
+* Istio integration has been updated to release 1.5.2 (Backport PR #12356, Upstream PR #11280, @jrajahalme)
+* Istio integration is simplified with Cilium build of istioctl. (Backport PR #12356, Upstream PR #10851, @jrajahalme)
+* Istio integration is updated to release 1.5.1, with backported fix for GKE/COS. (Backport PR #12356, Upstream PR #10730, @jrajahalme)
```

Comment:
I notice that `bugfixes` below has the 1.5.6 and 1.5.7 release updates, so probably makes sense to manually rearrange all of these istio fixes into the same section, make it a bit clearer to readers which one is the latest in the tree.